packages/nodes-base/nodes/Asana/Asana.node.ts

```diff
@@ -2248,6 +2526,45 @@ export class Asana implements INodeType {
 							responseData = responseData.data;
 						}
 					}
+
+					if (operation === 'update') {
+						// ----------------------------------
+						//        project:update
+						// ----------------------------------
+						const projectId = this.getNodeParameter('projectId', i) as string;
+						const additionalFields = this.getNodeParameter('additionalFields', i) as IDataObject;
+
+						// request parameters
+						requestMethod = 'PUT';
+						endpoint = `/projects/${projectId}`;
+
+						// optional parameters
+						if (additionalFields.color) {
+							qs.color = additionalFields.color;
+						}
+						if (additionalFields.due_on) {
+							qs.due_on = additionalFields.due_on;
+						}
+						if (additionalFields.name) {
+							body.name = additionalFields.name;
+						}
+						if (additionalFields.notes) {
+							qs.notes = additionalFields.notes;
+						}
+						if (additionalFields.owner) {
+							body.owner = additionalFields.owner;
+						}
+						if (additionalFields.team) {
+							body.team = additionalFields.team;
+						}
+						if (additionalFields.workspace) {
+							body.workspace = additionalFields.workspace;
```

Comment:
Workspace it;s not needed to update a project.

Reply:
Fixed in latest commit